packages/user_packages/README.md

```diff
@@ -16,8 +16,12 @@ System Properties which are pre-loaded in the registry during installation are r
 all the devices will internally inherit the the system properties from the registered edge system.
 The properties for the edge system can be set as 'key:value' pair, you can also set the location by passing the
 'latitude:value' and 'longitude:value' as properties in the user package.
+Optional Mechanism: If the set_properties or register call fails due to DCC_Comms publish exception
```

Comment:
The retry mechanism does not appear optional. It always happens and is hard-coded to 3 attempts? (And a 5 second sleep between attempts?)